package.json

```diff
@@ -85,7 +86,7 @@
     "lint:fix": "gulp eslint --fix",
     "coverage": "gulp coveralls",
     "generate-docs": "node ./build/commands-yml/parse.js",
-    "shrinkwrap-prod": "rimraf package-lock.json && npm prune --production && npm shrinkwrap && npm install --no-shrinkwrap",
+    "shrinkwrap:prod": "rimraf package-lock.json && rimraf node_modules && npm install --production && npm shrinkwrap && node ./check-npm-pack-files && npm install --no-shrinkwrap",
```

Comment:
I understand the `npm install --no-shrinkwrap` part from a developer convenience standpoint. But the script does more that the name lead one to believe. 

Say someone wants to package up a production zip and think they can use `shrinkwrap:prod` then they will be surprised to by the last step.

Reply:
@john7doe if somebody was to make a build from the git repo, you wouldn't need to concern yourself with creating a shrinkwrap file. The shrinkwrap file tells NPM what to lock the dependencies to install when Appium is installed globally or as an externally dependency. When `npm install` is run from within the repo, the shrinkwrap is ignored. So the best practice for creating a bundle is to simply run `npm install --production` to get production-only dependencies.